app/resources/cdl/charge_manager.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+# Controlled Digital Lending
+module CDL
+  class UnavailableForCharge < StandardError; end
+  class ChargeManager
+    attr_reader :resource_id, :eligible_item_service, :change_set_persister
+    # TODO: default eligible_item_service from #4033
+    def initialize(resource_id:, eligible_item_service:, change_set_persister:)
+      @resource_id = resource_id
+      @eligible_item_service = eligible_item_service
+      @change_set_persister = change_set_persister
```

Comment:
I will update the name on my Pr to be EligibleItemService

Reply:
Can we call it EligibleCdlService ? or just CdlService ?

Reply:
Never mind since CDL is there it would be redundant to include the Cdl in the class name. I'm ok with EligibleItemService.

Reply:
@christinach the name of the class doesn't matter, and does not need to change -- I think yours is more descriptive. It's just the method name and parameter signature we need to agree on. From the perspective of this class, it will be an "eligible item service" -- that's just the variable where it will hold the reference to the class you're writing.

Reply:
Oh you already changed it -- it looks good that way too. Might as well let it be!!